zcash_client_sqlite/src/wallet/orchard.rs

```diff
@@ -180,10 +316,7 @@ pub(crate) fn get_orchard_nullifiers(
     let nullifiers = stmt_fetch_nullifiers.query_and_then([], |row| {
         let account = AccountId(row.get(1)?);
```

Comment:
We don't do anything with the `id` column here, so we should not fetch it above (to match `get_sapling_nullifiers`).